msmbuilder/tests/test_dependencies.py

```diff
@@ -0,0 +1,27 @@
+import pip
+import os
+import warnings
+
+def test_installed_packages():
+    installed_packages = pip.get_installed_distributions()
+    package_names = [package.project_name for package in installed_packages]
+
+    test_dependencies = ['munkres', 'numdifftools', 'statsmodels', 'hmmlearn', 'cvxpy']
+
+    for td in test_dependencies:
+        if td not in package_names:
+            raise RuntimeError('Please install {} to continue'.format(td))
+
+def test_msmbuilder_data():
+    home = os.path.expanduser('~')
+    try:
+        os.stat(home+'/msmbuilder_data/')
```

Comment:
Use [`os.path.join`](https://docs.python.org/3/library/os.path.html#os.path.join) instead of adding strings.